pokete_data/map_data.py

```diff
@@ -2179,7 +2180,8 @@
             "tree_group_1": {
                 "x": 0,
                 "y": 0,
-                "txt":"""))())())())))()()())))((())(()()()))()))())(((()())())()()()()())))())))))(()(()()()()))((()  #    #  (((()))()((()()(()))((()()())((()()))()()))())))
+                "txt": """
```

Comment:
This also produces a CoordinateError.

Reply:
Fixed in ee3504b7128cc58e2dcc0a451993ace913c5fc08